internal/lookout/ui/make-test-jobs.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
It might better to just use load tester we have instead of this script. (https://github.com/G-Research/armada/tree/master/cmd/armada-load-tester, input https://github.com/G-Research/armada/blob/master/example/demo.yaml). We definitely lack documentation in this place, sorry.